cdk/domino_cdk/util.py

```diff
@@ -146,3 +149,9 @@ def overlay(alpha: dict, omega: dict) -> dict:
             return {}
         base_dict = check_type(dictionaries[0])
         return base_dict if len(dictionaries) == 1 else overlay(base_dict, cls.deep_merge(*dictionaries[1:]))
+
+    @staticmethod
+    def ruamel_dump(data: dict):
+        buf = BytesIO()
+        YAML().dump(data, buf)
```

Comment:
Ruamel is, uhh, "eccentric" about not dumping to anything but a stream. Hence, wrapper.